wagtailio/showcase/migrations/0003_add_listing_meta_description.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.2.18 on 2024-01-03 15:37
```

Comment:
nitpick: can this be part of the initial migration?

Reply:
That's a fair ask, I generally only do this once for the initial review, and subsequently add new migrations so the reviewer does not have to handle their DB state as much.